### ISSUE TITLE
:loading: option for "image" directive

### ISSUE THREAD

**Issue by h1nk** [sphinx-doc/sphinx#6706]
Lazy image loading is coming to the web soon (see whatwg/html#3752)! I think it would be very nice if Sphinx was able to support this web feature. This would be especially beneficial for documentation pages that are populated with lots of image content and large multi-megabyte vector graphics etc.

A simple `:loading:` option for the ["image" directive](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#images) that corresponds to the `loading` attribute of the `<img>` HTML element would need to be added in order to fulfill this feature.

**Comment by tk0miya** [comment]
How does the option effect to other formats? If my understanding is correct, it is available only for HTML outputs. -1 for adding new option for specific formats.

**Comment by h1nk** [comment]
Yes this would only be for HTML outputs like `html`, `dirhtml`, `singlehtml`, `json` etc. As I understand though there are already some `image` directive options that are only applicable to HTML output like `:alt:` and `:align:`.

**Comment by tk0miya** [comment]
No, `:alt:` and `:align:` have effected to other formats. For example, text and manpage builders uses `:alt:` option to render a text instead of the image. And LaTeX builder uses `:align:` option to layout the image in the output.
